src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php

```diff
@@ -83,6 +92,9 @@ public function testDispatchEvent()
         $this->assertSame(0, $subscriber2->calledByEventNameCount);
     }
 
+    /**
+     * @group legacy
```

Comment:
Test cases with this annotation are meant to be removed in the next major
Here, the test case mixes legacy and non-legacy use cases.
We need to split those so that we can safely remove all legacy test cases without reducing the coverage of the non-legacy features.

Reply:
Oops indeed 😅 I clean that up